atla5/resume
# develop: add tests, travis-ci; resolve export issues; update readme/json; refactor; output

### Description of changes:
- export: 
  - resolve errors with exporting the `Resume*` output files stemming from failing to `close()` files
  - remove commented out code for cleaning up excess `LaTeX` output files
- tests: 
  - lint `*.json` for any errors blocking their `json.loads()` 
  - test `humanize_*()` helper functions
  - test end-to-end running (without errors) and new dataset creation on `build_resume`
- travis:
  - hook these tests up to Travis-CI via new `.travis.yml` file
- readme:
  - update dependencies and add basic run instructions
  - add brief section on how to use the repo to make your own resume
- [refactor]:
  - move `build_resume.py` into build to keep everything in one place.
  - move all the `.json` and `.csv` into `/data`
- develop (misc):
  - remove extra newlines in output `Resume*.tex` file via `[file].write()`
  - various updates to json file contents and tweaking output pdf
- output: 2017-10-31
  
_note: this one's pretty huge. it should be smaller (broken up into two after the test/travis stuff), i just got excited._
